queue_services/entity-emailer/src/entity_emailer/email_templates/NR-REFUND.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <base href="/">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="referrer" content="origin-when-cross-origin"/>
+    <meta name="author" content="BC Registries and Online Services">
+    <title>Name Request notification from the Business Registry</title>
+    [[style.html]]
+  </head>
+
+  <body>
+    <table class="body-table" role="presentation">
+      <tr>
+        <td>
+
+          <div class="container">
+
+            [[whitespace-16px.html]]
+            <p>
+              This is confirmation that your name request {{ nr_number }} has been cancelled and will not be reviewed
+              by staff.
+            </p>
+
+            [[whitespace-16px.html]]
+            <p><strong>If you paid by credit card or a BC OnLine account:</strong></p>
+            <p>
+              A refund of <strong>${{ refund_value }} CAD</strong> (total payment value
+              including service fee) will be applied to your original payment method. Refunds are usually processed
+              within 10 to 14 business days.
+            </p>
+
+            [[whitespace-16px.html]]
+            <p><strong>If you paid by Pre-authorized Debit (PAD):</strong></p>
+            <p>
+              If the name request was cancelled on the day of the original purchase and prior to 7:00:00 PM Pacific
+              Time, you will not be charged. If the NR was cancelled after 7:00:00 PM Pacific Time on the day of the
+              purchase, you will recieve a refund to your PAD account within 10 to 14 business days.
```

Comment:
According to https://www2.gov.bc.ca/gov/content/governments/services-for-government/service-experience-digital-delivery/web-content-development-guides/web-style-guide/writing-guide/numbers the time format should be `7:00 pm Pacific time`